nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,110 @@
 
-## Finding the library
+## Microsoft Teams Live Share
 
-The TeamsJS client is distributed as an npm package. The latest version can be found here:
+The [Live Share SDK](https://github.com/microsoft/live-share-sdk) is compromised of three sub-packages with features to help developers create more collaborative moments across Microsoft Teams. Live Share (or sometimes called Live Share core) is for connecting to Fluid Framework through `sidePanel` and `meetingStage` for in meeting pop-up cards. Live Share media allows for media synchronization for HTML `<video>` and `<audio>` elements; which are beneficial for presenting video and audio is a Teams session. Live Share canvas adds annotation tools for whiteboard and PowerPoint presentations. 
 
-www.npmjs.com/package/@microsoft/teams-js
+### Finding the library
 
-## Installing the library
+The Live Share SDK is a JavaScript package comprised of three sub-packages ([Live Share](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md), [Live Share media](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-media-capabilities.md), and [Live Share canvas](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md)) which are published on npm and can be installed using npm or yarn.
 
-You can install the package using npm or yarn:
+To install the latest version of the Live Share packages to your application:
 
-* `npm install --save @microsoft/teams-js`
-* `yarn add @microsoft/teams-js`
+#### Live Share core
 
-## Using the library
+# [NPM](#tab/npm)
 
-If you are using any dependency loader or module bundler such as [RequireJS](http://requirejs.org/), [SystemJS](https://github.com/systemjs/systemjs), [browserify](http://browserify.org/), or [webpack](https://webpack.github.io/), you can use `import` syntax to import specific modules. For example:
+```bash
+npm install @microsoft/live-share@next --save
+```
+# [Yarn](#tab/yarn)
+
+To install using yarn
+```bash
+yarn add @microsoft/live-share@next
+```
+
+---
+
+#### Live Share media
 
-```typescript
-import * as microsoftTeams from "@microsoft/teams-js";
+# [NPM](#tab/npm)
+
+```bash
+npm install @microsoft/live-share@next --save
+npm install @microsoft/live-share-media@next --save
+```
+# [Yarn](#tab/yarn)
+
+```bash
+yarn add @microsoft/live-share@next
+yarn add @microsoft/live-share-media@next
 ```
 
-You can also reference the entire library in html pages using a script tag.  There are three ways to do this:
+---
 
-> [!IMPORTANT]
-> Do not copy/paste these `<script src=...` URLs from this page; they refer to a specific version of the SDK. To get the `<script src=...></script>` markup for the latest version, always go to https://www.npmjs.com/package/@microsoft/teams-js.
+#### Live Share canvas
```

Comment:
Suggestion for description:

Using Live Share canvas, your app can bring the power of [PowerPoint Live](https://support.microsoft.com/en-us/office/present-from-powerpoint-live-in-microsoft-teams-28b20e74-7165-499c-9bd4-0ad975d448ad) inking tools with minimal effort. For more, see [Live Share canvas overview](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-live-share-canvas)